src/components/avatars/OcAvatar.vue

```diff
@@ -1,9 +1,24 @@
 <template>
+  <img
+    v-if="src.length > 0"
```

Comment:
There is an issue: if the image fails to load we also need to fall back to the avatar component.